bindings/javascript/src/lib.rs

```diff
@@ -317,8 +413,13 @@ impl Statement {
     }
 
     #[napi]
-    pub fn raw() {
-        todo!()
+    pub fn raw(&mut self, raw: Option<bool>) {
```

Comment:
Can you document what these modes are?

Reply:
@jussisaurio Sure, but not here. The interface follows `better-sqlite3` (except its asynchronous), which defines the semantics.


@el-yawd worth patching bindings/javascript/docs/api.md